teamwork/templates/core/view_matches.html

```diff
@@ -10,9 +10,7 @@
   <link rel="stylesheet" href="{% static 'css/Tooltip.css' %}">
 {% endblock stylesheets %}
 
-{% block crumb %}
-<li><a href="{% url 'view_matches' %}">My Matches</a></li>
-{% endblock crumb %}
+
```

Comment:
Breadcrumb for My Matches should display: "Home > My Matches (active)". Currently only displays "Home"